### ISSUE TITLE
New side texture for shulker.

### ISSUE THREAD

**Issue by lipki** [SummerFields/MCPE#2]
2: add a descriptive title
3: if necessary add an explanation
In the MCPE version, I added a side texture to shulker! In this case, it will be reasonable display in the backpack!

4: adds images by drag and drop
You have to put the right textures directly
And in addition a screenshot if you want.

![shulker_side_black](https://cloud.githubusercontent.com/assets/401965/25844170/f60dda46-34a9-11e7-809d-7aa08abdfa49.png)![shulker_side_blue](https://cloud.githubusercontent.com/assets/401965/25844175/f6227b36-34a9-11e7-8f9e-6adb6138bd85.png)![shulker_side_brown](https://cloud.githubusercontent.com/assets/401965/25844171/f6164e42-34a9-11e7-9da3-183c7dc78291.png)![shulker_side_cyan](https://cloud.githubusercontent.com/assets/401965/25844172/f617db90-34a9-11e7-87b0-d154e0eb53d5.png)![shulker_side_gray](https://cloud.githubusercontent.com/assets/401965/25844174/f61fabcc-34a9-11e7-91e8-6dcc7637b5da.png)![shulker_side_green](https://cloud.githubusercontent.com/assets/401965/25844173/f61cf15c-34a9-11e7-97f9-cd78786ba91c.png)![shulker_side_light_blue](https://cloud.githubusercontent.com/assets/401965/25844176/f62ae5c8-34a9-11e7-8854-f3f357459088.png)![shulker_side_lime](https://cloud.githubusercontent.com/assets/401965/25844177/f6363810-34a9-11e7-9df4-c504f50acfa8.png)![shulker_side_magenta](https://cloud.githubusercontent.com/assets/401965/25844178/f639a680-34a9-11e7-9b8f-4aa7a51aa490.png)![shulker_side_orange](https://cloud.githubusercontent.com/assets/401965/25844179/f63eb968-34a9-11e7-92e7-f06d51357ce7.png)![shulker_side_pink](https://cloud.githubusercontent.com/assets/401965/25844180/f643bd82-34a9-11e7-9e4f-59a427438099.png)![shulker_side_purple](https://cloud.githubusercontent.com/assets/401965/25844182/f643d286-34a9-11e7-95bc-5587ef43c11a.png)![shulker_side_red](https://cloud.githubusercontent.com/assets/401965/25844181/f645596c-34a9-11e7-8db6-236124caa36a.png)![shulker_side_silver](https://cloud.githubusercontent.com/assets/401965/25844183/f6568264-34a9-11e7-8d70-e76a46e75461.png)![shulker_side_white](https://cloud.githubusercontent.com/assets/401965/25844184/f657d83a-34a9-11e7-94a4-54d6b29b1adf.png)![shulker_side_yellow](https://cloud.githubusercontent.com/assets/401965/25844185/f65b3476-34a9-11e7-894b-6ac69520356c.png)
![shulker_top_black](https://cloud.githubusercontent.com/assets/401965/25844186/f6614ce4-34a9-11e7-878e-3b196cb35f8d.png)![shulker_top_blue](https://cloud.githubusercontent.com/assets/401965/25844188/f66d7c08-34a9-11e7-8026-72b13a83523a.png)![shulker_top_brown](https://cloud.githubusercontent.com/assets/401965/25844187/f665bf68-34a9-11e7-92c7-1da3a4ba8d09.png)![shulker_top_cyan](https://cloud.githubusercontent.com/assets/401965/25844189/f673d7f6-34a9-11e7-9ab3-5dfaddc6cfc3.png)![shulker_top_gray](https://cloud.githubusercontent.com/assets/401965/25844190/f6749b8c-34a9-11e7-8db9-9d00fc8a6f40.png)![shulker_top_green](https://cloud.githubusercontent.com/assets/401965/25844191/f67cfe12-34a9-11e7-93a2-4ceea295ba03.png)![shulker_top_light_blue](https://cloud.githubusercontent.com/assets/401965/25844192/f67eaea6-34a9-11e7-822f-6ddf5e170828.png)![shulker_top_lime](https://cloud.githubusercontent.com/assets/401965/25844193/f6831b30-34a9-11e7-9876-ec08bb6f1aca.png)![shulker_top_magenta](https://cloud.githubusercontent.com/assets/401965/25844194/f68ed38a-34a9-11e7-9c75-9fa5ce02bb48.png)![shulker_top_orange](https://cloud.githubusercontent.com/assets/401965/25844195/f69042a6-34a9-11e7-8a80-7cf8ebdb9574.png)![shulker_top_pink](https://cloud.githubusercontent.com/assets/401965/25844196/f6929cc2-34a9-11e7-88f3-e43e9e313792.png)![shulker_top_purple](https://cloud.githubusercontent.com/assets/401965/25844197/f69bac22-34a9-11e7-9823-1d1f7b58fc67.png)![shulker_top_red](https://cloud.githubusercontent.com/assets/401965/25844198/f69ea008-34a9-11e7-8719-6b562f13b467.png)![shulker_top_silver](https://cloud.githubusercontent.com/assets/401965/25844199/f6a47f0a-34a9-11e7-8ded-fa3c16c3fecf.png)![shulker_top_white](https://cloud.githubusercontent.com/assets/401965/25844200/f6ab9ace-34a9-11e7-9cda-80f8a72b829e.png)![shulker_top_yellow](https://cloud.githubusercontent.com/assets/401965/25844201/f6b08d2c-34a9-11e7-951e-90f0ef089746.png)

**Comment by chengzzi89** [comment]
Are you teaching me how to upload a texture map here? It seems that mobile devices can not perform it!😩

**Comment by lipki** [comment]
You can switch the site to desktop version.
The button is all at the bottom of each page.
In desktop mode, there is a link here ⬇(bottom in edit mode) to add an image.

**Comment by tobyplowy** [comment]
@chengzzi89 also Before you add anything please take a look at the normal mc pack to see if the texture already exists (so you don't have to do extra work and all the texturepack versions are consistent)

**Comment by chengzzi89** [comment]
@tobyplowy No, it isn't！ I just made several improvements to the MCPE version of the GUI texture!😟

**Comment by lipki** [comment]
Do not worry, what you do looks fine, but you'll have to share it here.
Can you put a picture?

**Comment by tobyplowy** [comment]
@chengzzi89 Please post pictures

**Comment by chengzzi89** [comment]
Okay, I'll try!😃😊

**Comment by chengzzi89** [comment]
I have a question! Why doesn't SummerFields have sounds? I think we should have this project!

**Comment by lipki** [comment]
make a new issue for that.

**Comment by tobyplowy** [comment]
@chengzzi89 and if you make that issue do it here please https://github.com/SummerFields/SummerFields/issues
